packages/elements/src/colors/index.ts

```diff
@@ -29,6 +29,7 @@ export const colors = {
     CIVIL_GRAY_4: "#E9E9EA",
     CIVIL_GRAY_5: "#F4F4F4",
     CIVIL_GRAY_6: "#FCFCFC",
+    CIVIL_GRAY_7: "#9B9B9B", // NewPrimaryButton border color
```

Comment:
Aw man now the grays aren't numbered in increasing lightness, but I guess no way around it unless `CIVIL_GRAY_2A` or something, which would be dumb.